newfiles_skel/etc/cups/policy_nego.conf

```diff
@@ -1,4 +1,4 @@
-<Policy authenticated>
+<Policy negotiate>
   # Job/subscription privacy...
```

Comment:
This new oppolicy does not contains an "AuthType Negotiate" statement. It only has got "Negotiate" in its name, which is unuseful.